include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -241,6 +241,15 @@ class CommonOperatorLoader : public ProtobufLoader {
     nodeValueByName_[op.output(0)] = NodeValue(in, 0);
   }
 
+  void loadCast(const OpType &op, ArgumentDictionaryTy &dict) {
```

Comment:
Does Cast imply a possible reshape?

Reply:
[Cast only changes the type](https://caffe2.ai/docs/operators-catalogue.html#cast)